content/docs/start/experiments.md

```diff
@@ -31,16 +32,15 @@ $ dvc exp run --set-param featurize.max_features=3000
 
 <details>
 
-### 💡 Expand to see what this command does.
+### 💡 Expand to see what happens under the hood.
 
 `dvc exp run` is similar to `dvc repro` but with some added conveniences for
 running experiments. The `--set-param` (or `-S`) flag sets the values for
-[parameters](/doc/command-reference/params) as a shortcut to editing
-`params.yaml`.
+<abbr>parameters<abbr> as a shortcut for editing `params.yaml`.
 
 Check that the `featurize.max_features` value has been updated in `params.yaml`:
 
-```git
+```diff
```

Comment:
is this another `diff` to be reverted? Please

- check the handful of places in the docs where `diff` is used
- fix the indentation issue with both `git` & `diff`

Reply:
Yep, please revert for now.

Reply:
done